csharp/src/Microsoft.ML.OnnxRuntime/InferenceSession.shared.cs

```diff
@@ -136,6 +136,17 @@ public InferenceSession(byte[] model, SessionOptions options)
         {
             Init(model, options);
         }
+        
+        /// <summary>
```

Comment:
I feel this should not be a part of the change. Let' make another PR

Reply:
I keep the ctor but made it `internal` so it still can be part of the unit test suit. Will expand on the constructors in another PR.